routes/index.js

```diff
@@ -40,6 +40,13 @@ router.get('/conversation', user.redirectLogIn, (req, res, next) => {
     res.render('conversation', { page_title: req._("Almond - Chat") });
 });
 
+router.get('/conversation-beta', user.redirectLogIn, (req, res, _) => {
+    res.render(
+        'conversation_beta', 
+        { page_title: req._("Almond - Chat (Experimental)") }
```

Comment:
We're not running AB tests, and we're not maintaining 2 chat interfaces at the same time to confuse people. Kill the experimental label and make this the default interface.

Reply:
Ok